efficientdet/keras/utils_keras.py

```diff
@@ -18,7 +18,14 @@
 
 from typing import Text
 import tensorflow as tf
-import utils
+
+
+def batch_norm_class(is_training, strategy=None):
+  if is_training and strategy == 'horovod':
+    return tf.keras.layers.experimental.SyncBatchNormalization
```

Comment:
I don't understand.

Reply:
I got warning on tf.add_to_collection from tensorflow, so to fix it I added this function